example/synchronous_lstm/imdb.py

```diff
@@ -77,6 +77,7 @@ def get_dataset_file(dataset, default_dataset, origin):
 
 
 def load_data(path="imdb.pkl", n_words=100000, valid_portion=0.1, maxlen=None,
+
```

Comment:
Remove the empty line here. Maybe rebase at the same time to remove the merge commit.

Reply:
Sure. I'll squash everything into one.